modules/accounting/assets/src/admin/components/list-table/ListTable.vue

```diff
@@ -131,10 +131,10 @@
                         </div>
                     </th>
                     <td v-for="(value, key) in columns" :key="key" :data-colname="ucFirst(key)"
-                        :class="['column', key, (value.isColPrimary) ? 'column-primary' : '', (actionColumn === key && hasActions) ? 'col--actions': '', { 'selected': checkedItems.includes(row[index]) }]">
+                        :class="['column', key, (value.isColPrimary) ? 'column-primary' : '', /*(actionColumn === key && hasActions) ? 'col--actions': ''*/, { 'selected': checkedItems.includes(row[index]) }]">
```

Comment:
Why you are removing the `col--actions` selector? If it is not needed remove it instead of commenting out like this.

Reply:
because of this class, the actions buttons (three vertical dots) on a small screen were becoming hidden.